route/build/src/popMetadat.f90

```diff
@@ -136,6 +136,7 @@ subroutine popMetadat(err,message)
  meta_NTOPO  (ixNTOPO%isLakeInlet    ) = var_info('isLakeInlet'    , 'flag to define if a lake inlet (1=true)'            ,'-'    ,ixDims%seg   , .false.)
  meta_NTOPO  (ixNTOPO%userTake       ) = var_info('userTake'       , 'flag to define if user takes water (1=true)'        ,'-'    ,ixDims%seg   , .false.)
  meta_NTOPO  (ixNTOPO%goodBasin      ) = var_info('goodBasin'      , 'flag to define a good basin (1=true)'               ,'-'    ,ixDims%upSeg , .false.)
+ meta_NTOPO  (ixNTOPO%islake         ) = var_info('islake'         , 'flag to define if the object is lake (1=true)'      ,'-'    ,ixDims%upSeg , .false.) ! has added the lake flag
```

Comment:
Ok,  `ixDims%upSeg` is not correct.    it should be `ixDims%seg`.   isLake is not information for upstream reach, but for current reach.

Reply:
compiles and runs but still does not enter to the lake routing as if all the segments are false...